packages/react/src/components/Card/Card.stories.tsx

```diff
@@ -50,113 +47,48 @@ Preview.args = {
-            target='_blank'
-            rel='noopener noreferrer'
-          >
-            Card Neutral
-          </a>
-        </Heading>
-        <Paragraph>
-          Most provide as with carried business are much better more the
-          perfected designer. Writing slightly explain desk unable at supposedly
-          about this
-        </Paragraph>
-      </Card.Block>
-    </Card>
-    <Card data-color='subtle'>
-      <Card.Block>
-        <img src={cat2} alt='katt' />
-      </Card.Block>
-      <Card.Block>
-        <Heading>
-          <a
-            href='https://designsystemet.no'
-            target='_blank'
-            rel='noopener noreferrer'
-          >
-            Card Subtle
-          </a>
-        </Heading>
-        <Paragraph>
-          Most provide as with carried business are much better more the
-          perfected designer. Writing slightly explain desk unable at supposedly
-          about this
-        </Paragraph>
-      </Card.Block>
-    </Card>
-    <Card data-color='brand1'>
-      <Card.Block>
-        <img src={cat3} alt='katter' />
-      </Card.Block>
-      <Card.Block>
-        <Heading>
-          <a
-            href='https://designsystemet.no'
-            target='_blank'
-            rel='noopener noreferrer'
-          >
-            Card First
-          </a>
-        </Heading>
-        <Paragraph>
-          Most provide as with carried business are much better more the
-          perfected designer. Writing slightly explain desk unable at supposedly
-          about this
-        </Paragraph>
-      </Card.Block>
-    </Card>
-    <Card data-color='brand2'>
-      <Card.Block>
-        <img src={cat4} alt='katt' />
-      </Card.Block>
-      <Card.Block>
-        <Heading>
-          <a
-            href='https://designsystemet.no'
-            target='_blank'
-            rel='noopener noreferrer'
-          >
-            Card Second
-          </a>
-        </Heading>
-        <Paragraph>
-          Most provide as with carried business are much better more the
-          perfected designer. Writing slightly explain desk unable at supposedly
-          about this
-        </Paragraph>
-      </Card.Block>
-    </Card>
-    <Card data-color='brand3'>
-      <Card.Block>
-        <img src={cat5} alt='katt' />
-      </Card.Block>
-      <Card.Block>
-        <Heading>
-          <a
-            href='https://designsystemet.no'
-            target='_blank'
-            rel='noopener noreferrer'
-          >
-            Card Third
-          </a>
-        </Heading>
-        <Paragraph>
-          Most provide as with carried business are much better more the
-          perfected designer. Writing slightly explain desk unable at supposedly
-          about this
-        </Paragraph>
-      </Card.Block>
-    </Card>
+    {Object.entries(VariantsMap).map(([key, value]) => (
+      <Card key={key} {...value}>
+        <Card.Block>
+          <Paragraph>{key}</Paragraph>
```

Comment:
Maybe change this to print `data-color` and `data-variant` instead of key?

For example just:
```
data-color: 'info'
data-variant: 'tinted'
```

Reply:
![image](https://github.com/user-attachments/assets/db6bc350-209c-42c4-87de-c04f20d3b1ec)